Phoenix/PHAppDelegate.m

```diff
@@ -89,6 +89,14 @@ - (IBAction) reloadContext:(id)__unused sender {
     [self.context load];
 }
 
+- (IBAction) editConfig:(id)__unused sender {
+    NSWorkspace *workspace = [NSWorkspace sharedWorkspace];
+    NSString * path    = self.context.primaryConfigurationPath;
+
+    [workspace openFile: path];
```

Comment:
Leave one empty line to the beginning of the method block. Also you can simplify this to one line with `[[NSWorkspace sharedWorkspace] openFile:self.context.primaryConfigurationPath]`.